docs/source/guide/shadows-5-theory.md

```diff
@@ -98,7 +102,7 @@ On the other hand, a random Pauli measurement (11) means that for each qubit, we
 \end{equation}
 
 
-The Clifford measurement requires the depth of the circuit to grow linearly with system size, which is not currently feasible for large systems, so we are going to implement the local (Pauli) measurement and integrate it into Mitiq in the current stage. However, it is worth noting that there is an intermediate step of scrambling the circuits and combining the local and global measurement {cite}`hu2023classical`. 
+The Clifford measurement requires the depth of the circuit to grow linearly with system size, which is not currently feasible for large systems, which is why only the local (Pauli) measurement is implemented in Mitiq in the current stage. However, it is worth noting that this method involves an intermediate step of scrambling the circuits and combining the local and global measurement {cite}`hu2023classical`. 
```

Comment:
I made these changes here because the wording makes it sound like work on `mitiq.shadows` is in the planning stage.

Reply:
Thanks for catching this!